src/main/steps/applicant1/hub-page/right-column/content.ts

```diff
@@ -64,7 +64,10 @@ const languages = {
 };
 
 export const generateContent: TranslationFn = content => {
-  const aosSubmitted = !content.isJointApplication && content.userCase.applicant2IBelieveApplicationIsTrue;
+  const aosSubmitted =
+    !content.isJointApplication &&
+    (content.userCase.applicant2IBelieveApplicationIsTrue ||
+      content.userCase.documentsUploaded?.find(doc => doc.value.documentType === DocumentType.RESPONDENT_ANSWERS));
```

Comment:
I don't think RESPONDENT_ANSWERS can be generated without applicant2IBelieveApplicationIsTrue being set so I don't think that extra check is necessary